webpack.config.js

```diff
@@ -119,7 +115,8 @@ module.exports = (_env, options) => ({
                 ]
               }
             }
-          }
+          },
+          { test: /\.cbor$/, use: "null-loader" }
```

Comment:
Can you tell me what the purpose of this line is? It's for the externs files produced by the compiler, right? I wouldn't have expected webpack to try to load these files at all unless something else tries to `require` them.

Reply:
I don't know why they were being required (I couldn't find anything importing them) and the app still worked, but the terminal and browser console were both filled with warnings for every cbor file. The whole output folder does show up in the browser "sources" tab, maybe some part of webpack's sourcemap feature was trying to read them? I can look into it more if this seems hacky.

Reply:
It doesn't seem worth worrying that much about to me, I was just wondering.

Reply:
I did see other `cbor` files in `node_modules`, and webpack supports wasm.. maybe it has a use in JS-world I'm not aware of.